dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -594,5 +594,11 @@ KERBEROS_NOT_SUPPORTED_WARNING.explanation= The Application Server does not supp
 KERBEROS_NOT_SUPPORTED_WARNING.useraction= Disable Kerberos, or use a DB2 backend with the DB2 universal JDBC driver.
 
 # 9544-9548 deleted
+
+# {0} name of the DataSource
+INVALID_LOGINTIMEOUT=DSRA9549E: The Application Server does not support the loginTimeout property specified in the URL or properties of the {0} DataSource.
+INVALID_LOGINTIMEOUT.explanation=The Application Server does not support the loginTimeout property when specified for a DataSource that uses the Driver type.
+INVALID_LOGINTIMEOUT.useraction=Remove the loginTimeout property or modify the DataSource configuration to use a DataSource type rather than a Driver type.
+
 # 9600-9603 deleted 
```

Comment:
This message was placed in the wrong section of the JDBC messages file.  This messages file has different numeric ranges depending on whether messages came from traditional WebSphere code or whether they are from Open Liberty.  Open Liberty messages belong in the range 4000-4999.